nsxt/resource_nsxt_policy_group.go

```diff
@@ -821,6 +824,7 @@ func getIdentityGroupsData(expressions []*data.StructValue) ([]map[string]interf
 }
 
 func validateGroupCriteriaAndConjunctions(criteriaSets []interface{}, conjunctions []interface{}) ([]criteriaMeta, error) {
+	log.Printf("[INFO] ##### CRITERIA SET: %v, CONJUNCTIONS: %v", len(criteriaSets), len(conjunctions))
```

Comment:
Did you plan on leaving this? Or is it a debugging message?

Reply:
It's a debugging message. Thankfully I only left this one :)

Reply:
fixed